apps/emqx_bridge/src/emqx_bridge.erl

```diff
@@ -75,6 +72,16 @@
 
 load() ->
     Bridges = emqx:get_config([bridges], #{}),
+    %% When using Hocon aliases, the raw config still holds the legacy
```

Comment:
It is not quite clear why we should do this: it looks like hocon alias abstraction leakage 🤔  

>the raw config still holds the legacy type name
Isn't this always a case when hocon aliases are used? 🤔

Reply:
I mean, since aliases are a publicly available feature, any contributor may start to use them in the schemas. Trying to maintain some level of "raw config consistency" everywhere would be a nightmare 😓 
Shouldn't we better discourage raw config usage?

Reply:
I suspect it's related to this (still looking into it):

https://github.com/emqx/emqx/blob/567413389cdd3e33354508ede0e7aef3a24526d1/apps/emqx_bridge/src/emqx_bridge.erl#L272